packages/flutter_tools/lib/src/localizations/gen_l10n.dart

```diff
@@ -177,6 +177,57 @@ String generateNumberFormattingLogic(Message message) {
   return formatStatements.isEmpty ? '@(none)' : formatStatements.join();
 }
 
+/// To make it easier to parse plurals or select messages, temporarily replace
+/// each "{placeholder}" parameter with "#placeholder#" for example.
+String _replacePlaceholdersBraces(
+  String translationForMessage,
+  Iterable<Placeholder> placeholders,
+  String replacementBraces,
+) {
+  assert(replacementBraces.length == 2);
+  String easyMessage = translationForMessage;
+  for (final Placeholder placeholder in placeholders) {
+    easyMessage = easyMessage.replaceAll(
+      '{${placeholder.name}}', '${replacementBraces[0]}${placeholder.name}${replacementBraces[1]}');
```

Comment:
nit: opening and closing parenthesis should have same indentation level:

```suggestion
    easyMessage = easyMessage.replaceAll(
      '{${placeholder.name}}', 
      '${replacementBraces[0]}${placeholder.name}${replacementBraces[1]}',
    );
```

Reply:
fixed